test/tests/neml_simple/tests

```diff
@@ -6,14 +6,39 @@
     csvdiff = 'le_stress_out.csv'
     required_objects = NEMLStress
     design = 'ComputeNEMLStress.md'
-    requirement = 'BlackBear shall be capable of using the NEML library to compute the response of a linearly elastic material'
+    requirement = 'BlackBear shall be capable of using the NEML library to compute the 
+                   response of a linearly elastic material parsed from xml'
+  [../]
+  [./neml_linear_elastic_variableOverwrite]
+    type = 'CSVDiff'
+    input = 'le_stress_variableOverwrite.i'
+    csvdiff = 'le_stress_out.csv'
+    required_objects = NEMLStress
+    design = 'ComputeNEMLStress.md'
```

Comment:
`ComputeNEMLStress.md` doesn't actually exist. I don't know why this stuff isn't failing the tests. I think it existed at one time, but then we renamed it to `NEMLStress`. If you could clean up the existing references to it while you're in here, that would be great.

Reply:
These should all be fixed.